src/main/java/com/example/petbuddybackend/service/photo/FirebasePhotoService.java

```diff
@@ -67,6 +70,30 @@ public PhotoLink uploadPhoto(MultipartFile multipartFile) {
         return photoRepository.save(photo);
     }
 
+    @Override
+    @Transactional
+    public List<PhotoLink> uploadPhotos(List<MultipartFile> multipartFiles) {
+        if(multipartFiles.isEmpty()) {
+            return Collections.emptyList();
+        }
+
+        multipartFiles.forEach(this::validatePhoto);
```

Comment:
w uploadPhoto juz i tak walidujemy zdjecie

Reply:
Faktycznie się zdublowało. Sprawdzanie na całej liście przed operacją i tak by sie przydało, ale jakoś to przepiszę żeby podwójnie nie było sprawdzane

Reply:
A nie, cofam. Tam używam `uploadFile` a nie jak myślałem `uploadPhoto`. W `uploadFile` nie ma walidacji, więc się nie dubluje. Nie trzeba więc nic zmieniać